assets/javascripts/discourse/initializers/apply-global-filter-styles.js

```diff
@@ -16,8 +16,8 @@ export default {
     }
 
     const globalFilters = siteSettings.global_filters.split("|");
-    const currentUser = container.lookup("current-user:main");
-    router.one("didTransition", () => {
+    const currentUser = container.lookup("service:current-user");
+    router.one("routeDidChange", () => {
```

Comment:
Does the router service have a `one` method?

Reply:
Yeah it's part of `Evented` https://api.emberjs.com/ember/release/classes/evented

Reply:
Too bad ember docs don't mention that the Router service has those methods/extends Evented 🙃